test/parallel/test-cluster-inspector-debug-port.js

```diff
@@ -13,7 +13,7 @@ if (cluster.isMaster) {
 
   function fork(offset, execArgv) {
     if (execArgv)
-      cluster.setupMaster({execArgv});
+      cluster.setupMaster({args: execArgv});
```

Comment:
@joshgav I think you'd have to connect to the debugger to move past the break. There are other tests that exercise `--debug-brk`, but I don't think any that do it with the cluster module. Maybe this would be better to move to a separate test.